pkgs/development/python-modules/playwright/default.nix

```
@@ -16,15 +16,15 @@ in
 buildPythonPackage rec {
   pname = "playwright";
   # run ./pkgs/development/python-modules/playwright/update.sh to update
-  version = "1.38.0";
+  version = "1.40.0";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```

and add `setuptools` into `nativeBuildInputs` instead. Enables the PEP517 build process, as opposed to running `python setup.py install`, which is deprecated.